tests/unit/data/observatory/enrichment/test_enrichment_service.py

```diff
@@ -35,7 +36,7 @@ def test_prepare_data(self):
         df = pd.DataFrame([[1, point]], columns=['cartodb_id', geom_column])
```

Comment:
We should use a GeoDataFrame

Reply:
I have changed this in my branch `1231-index-management`

Reply:
Cool! Could you update this issue  https://github.com/CartoDB/cartoframes/issues/1255?

Reply:
Sorry, I meant that I changed the tests. However, I have added a comment to that issue too.